docs/assets/center_img.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+/**
+ * Component for centering an image in the docs
+ *
+ * @param {*} props :
+ *  src: image source
+ *  alt: image alternative text
+ *  imgStyle: object, any image styles (props can be passed in as: `imgStyle={{ maxHeight: "600px", border: "solid" }}`)
+ * @returns a component for uniformally centering images in the doc
+ */
+
+export default function CenterImg(props) {
```

Comment:
This is an example of how you could set up the component with prop delegation

```suggestion
export default function CenterImg({src, alt, ...delegated}) {
```